lib/balance-tx/lib/Cardano/Tx/Balance.hs

```diff
@@ -1,7 +0,0 @@
--- |
--- Copyright: © 2022 IOHK
--- License: Apache-2.0
---
-module Cardano.Tx.Balance
-    ()
-    where
```

Comment:
I think @Unisay will be happy that this module has gone 😄

Reply:
Bye bye, you've been a good module!